src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/help.jelly

```diff
@@ -54,6 +54,77 @@ THE SOFTWARE.
     </p>
     <pre>stage name: 'Build', concurrency: 1</pre>
     <p>
-        There are also some predefined variables, shown at the bottom.
+        Many steps require complex nested configuration for a parameter.
+        (And some nested configuration objects in turn have object-typed parameters.)
+        There are three ways these nested objects may be specified, in decreasing order of preference:
+    </p>
+    <ol>
+        <li>
+            <p>
+                Some configuration objects define custom <em>symbols</em>.
+                These are used with a notation looking like a step call, or other function call taking a map of named parameters:
+            </p>
+            <pre>step archiveArtifacts('**.txt')</pre>
```

Comment:
Here is one way to fix this. Normal execution environment doesn't allow `archiveArtifacts` call being invoked here to tell apart whether there's a function call around it or not (`step` in this case), but since we run this in our own interpreter that is groovy-cps, we can actually tell it apart.

By using the call site tagging mechanism in https://github.com/cloudbees/groovy-cps/pull/36, we can mark those call sites that are at the statement level (aka top-level call), and change the behaviour of meta-step handling accordingly.